test/test.js

```diff
@@ -69,33 +69,30 @@ describe('onclickoutside hoc', function() {
 
 
   it('should call the specified handler when clicking the document', function() {
-    var Component = React.createClass({
-      getInitialState: function() {
-        return {
+    class Component extends React.Component {
```

Comment:
why not use a proper stateless component, which is just a `function(props) { return UICodeGoesHere }`?